lib/collection/proxy-config-list.js

```diff
@@ -0,0 +1,80 @@
+var _ = require('../util').lodash,
+    PropertyList = require('./property-list').PropertyList,
+    ProxyConfig = require('./proxy-config').ProxyConfig,
+
+    MATCH_ALL = '*',
+    MATCH_ALL_REGEX = '.*',
```

Comment:
Do you think we should use `.*?` here? `.*` is a greedy operator.
For example, if someone enters a pattern of `http://10.0.*.*/some-path`, the `.*` will match both the asterisks.

If this one, we should probably add a test, because it's very easy to break the behavior with a single character change.